views/index.html

```diff
@@ -2,20 +2,139 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <link rel="icon" type="image/x-icon" href="images/favicon.png">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
     <title>Index</title>
 </head>
+
 <body>
-    <header class="bg-dark text-light">
+    <header id="header" class="header">
```

Comment:
could you move the header and footer and any imports to a master template file that can be extended by this file and any subsequent ones without having to copy/paste all this code